src/main/java/coresearch/cvurl/io/model/CvurlConfig.java

```diff
@@ -27,7 +27,7 @@
 /**
  * Data class that holds configuration of {@link coresearch.cvurl.io.request.CVurl}
  */
-public class Configuration {
+public class CvurlConfig {
```

Comment:
CVurlConfig?

Reply:
yes

Reply:
I mean, it's better to name this class _CVurlConfig_ instead of _CvurlConfig_

Reply:
2much upper cases, no?